scholarx/2022/index.html

```diff
@@ -675,10 +675,11 @@ <h6>{{title}}</h6>
                                  class="avatar img-raised overflow-visible">
                             <span>{{author_name}}</span>
                         </div>
-                        <div class="stats ml-auto">
+                       <!-- <div class="stats ml-auto">
                             <i class="far fa-clock"></i> {{readingTime}} min read
-                        </div>
+                        </div>-->
                     </div>
+   
```

Comment:
Let's just remove this. @janithlahirukariyawasam

Reply:
Done

Reply:
No worries